src/error_estimation/jump_error_estimator.C

```diff
@@ -307,6 +318,11 @@ void JumpErrorEstimator::estimate_error (const System & system,
               fine_context->side_fe_reinit();
             }
 
+#ifdef LIBMESH_ENABLE_INFINITE_ELEMENTS
+          if (e->neighbor_ptr(n_e) ->infinite()) // view infinite elements as boundary; don't integrate boundary_sides.
+             continue;
+#endif // LIBMESH_ENABLE_INFINITE_ELEMENTS
+
```

Comment:
* this is the source of segfault: the `neighbor_ptr` may be `null`.
I just wonder why it didn't fail before...

Reply:
I think dereferencing a nullptr only technically results in undefined behavior, so it's not *guaranteed* to crash, although that is usually the result. Hard to say if that was at play for you here, though.